source/components/handling-events.md

```diff
@@ -0,0 +1,125 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
+
+For example, imagine we have a template like this:
+
+```hbs
+{{#x-clickable}}
+  This is a clickable area!
+{{/x-clickable}}
+```
+
+Let's implement `x-clickable` such that when it is
+clicked, an alert is displayed:
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    alert("XClickableComponent was clicked!");
+  }
+});
+```
+
+Browser events may bubble up the DOM which may target parent component(s) in
+succession. To enable bubbling `return true;` from the event handler method in
+your component.
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    Ember.Logger.info("XClickableComponent was clicked!");
+    return true;
+  }
+});
+```
+###Sending Actions
+
+To have the click event from `x-clickable` affect the state of your application,
+simply send an action to the component's `targetObject`.
+
+This example shows how to convert an event into an action:
+
+````app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    this.sendAction('turnItUp', 11);
+  }
+});
+````
+
+By default a component's `targetObject` is the template's controller instance.
```

Comment:
Oh so targetObject is private now, http://emberjs.com/api/classes/Ember.Component.html#property_targetObject oops.